SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -181,7 +185,7 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 			Components.Add(improbable::Position(PlaceholderPosition).CreatePositionData());
 			Components.Add(improbable::Metadata(TEXT("Placeholder")).CreateMetadataData());
 			Components.Add(improbable::Persistence().CreatePersistenceData());
-			Components.Add(improbable::EntityAcl(SpatialConstants::UnrealServerPermission, ComponentWriteAcl).CreateEntityAclData());
+			Components.Add(improbable::EntityAcl(ServerWorkerRequirementSet, ComponentWriteAcl).CreateEntityAclData());
```

Comment:
This might be conflicting with https://github.com/spatialos/UnrealGDK/pull/929

Reply:
Can do that in a separate PR to remove placeholder actors if necessary